tests/functional/aws-node-sdk/test/object/initiateMPU.js

```diff
@@ -65,5 +66,120 @@ describe('Initiate MPU', () => {
                 done();
             });
         });
+
+        describe('with tag set', () => {
```

Comment:
I'm thinking those tests could be shared between MPU and non-MPU by adding them in the lib/utility/tagging.js file?